src/pudl/metadata/fields.py

```diff
@@ -1874,6 +2009,80 @@
             "min, equal to or less than 5 min, or some other way."
         ),
     },
+    "monthly_average_consumption_rate_gallons_per_minute": {
+        "description": "Monthly average consumption rate of cooling water",
+        "type": "number",
+        "unit": "gpm",
+    },
+    "monthly_average_discharge_rate_gallons_per_minute": {
+        "description": "Monthly average discharge rate of cooling water - available 2010+",
```

Comment:
I'd drop the 'available 2010+' as this is true for a bunch of other columns but only reported here, no?